src/info/utils/mod.rs

```diff
@@ -120,6 +126,21 @@ mod tests {
         format_time(time, false);
     }
 
+    #[test]
+    fn test_timezone_awareness() {
+        let current_time = SystemTime::now()
+            .duration_since(SystemTime::UNIX_EPOCH)
+            .unwrap();
+        let hour_offset: i32 = 1;
+        let offset = 60 * 60 * hour_offset;
+        let also_now = Time::new(
+            (current_time.as_secs() as gix::date::SecondsSinceUnixEpoch) + i64::from(offset),
+            -offset,
+        );
+        let diff = diff_gix_time(SystemTime::now(), also_now).unwrap();
```

Comment:
I wouldn't repeat the `SystemTime::now()` call, but reuse the value from above. This is not deterministic.